source/documentation/deploying_services/configure_cdn.md

```diff
@@ -0,0 +1,87 @@
+## Set up a custom domain by configuring your own CDN
+
+This section explains how to configure a custom domain name for your application by configuring your own CDN.
+
+There are two ways to configure your own CDN:
```

Comment:
Do you think we should summarise the trade-offs between these two methods and the third `cdn-route` method?